.github/workflows/test-rust.yaml

```diff
@@ -79,7 +79,9 @@ jobs:
           runner.os == 'Linux') }}
         with:
           command: insta
-          args: test --unreferenced=auto ${{ inputs.target_option }}
+          args:
+            test --unreferenced=auto ${{ inputs.target_option }} --features
```

Comment:
Instead of using the env var, we could do something like `${{ inputs.os == 'ubuntu-latest' && '--features=test-external-dbs' || '' }}`

Reply:
sorry, let me add a suggestion